src/algebra/m231.rs

```diff
@@ -229,4 +267,12 @@ mod tests {
 
         TestResult::from_bool(x * x.invert() == Mod231(1))
     }
+
+    #[quickcheck]
+    fn prop_roundtrip_i32(x: Mod231) -> bool {
+        let y: i32 = x.into();
+        let z: Result<Mod231, _> = Mod231::try_from(y);
+        z == Ok(x)
```

Comment:
FWIW, in unit tests, it is possible to return a `Result` which saves the unwrapping step. I'll redo this test to demonstrate what I mean.